collect_app/src/main/java/org/odk/collect/android/activities/FormDownloadList.java

```diff
@@ -286,7 +276,7 @@ private void clearChoices() {
     @Override
     protected void onListItemClick(ListView l, View v, int position, long id) {
         super.onListItemClick(l, v, position, id);
-        mDownloadButton.setEnabled(!(selectedItemCount() == 0));
+        mDownloadButton.setEnabled(!(ListViewUtils.selectedItemCount(getListView()) == 0));
```

Comment:
Good point that especially in the absence of tests it makes sense to limit the number of changes made. Still, I'd prefer to see some of these smaller things that are easy to verify manually cleaned up while we have the code and its intent firmly in our minds! That will make the behemoth eventual refactor just a little bit easier/more pleasant. But if you don't mind, I'd like to get some more opinions in since this kind of thing will be coming up a lot. @yanokwa, @batkinson?